src/main/java/com/kramekk1/medicalClinic/PatientValidator.java

```diff
@@ -34,4 +34,10 @@ public static void validateNullField(Patient newPatient) {
             throw new IllegalArgumentException("One or more patient data field is null");
         }
     }
+
+    public static void validateIncorrectEmail(Optional<Patient> patient) {
+        if (patient.isEmpty()) {
+            throw new IllegalArgumentException("Entered email not exist");
```

Comment:
Moim zdaniem mozemy popracowac nad lepszym messagem. -> Wydaje mi sie ze to jest sytuacja nie kiedy on jest incorrect tylko kiedy nie bylo takiego pacjenta